code/parachain/frame/composable-traits/src/assets.rs

```diff
@@ -53,7 +56,7 @@ pub struct AssetInfo<Balance> {
 	/// Symbol of the asset.
 	pub symbol: Option<BiBoundedAssetSymbol>,
 	/// The number of decimals this asset uses to represent one unit.
-	pub decimals: u8,
+	pub decimals: Option<u8>,
```

Comment:
I still think we should use something like this for decimals: https://github.com/ComposableFi/composable/pull/2954#discussion_r1100288779